tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml

```diff
@@ -10,6 +10,10 @@ data:
   ES_PASSWORD: "changeme"
   ES_HOST: "elasticsearch-master.kube-system"
   USE_DOCKER: "false"
+  EKS_CONFIG_1_NODE_1: {{ .Values.testData.conf1Node1}}
+  EKS_CONFIG_1_NODE_2: {{ .Values.testData.conf1Node2}}
+  EKS_CONFIG_2_NODE_1: {{ .Values.testData.conf2Node1}}
+  EKS_CONFIG_2_NODE_2: {{ .Values.testData.conf2Node2}}
```

Comment:
Use `{{ range }}` for iteration
https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action

Reply:
`range` has been added to the `configmap`